dev/how-to/migrate/full-from-mysql.md

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
Recommend using the file name "full-data-from-mysql.md", so the directory will be `dev/how-to/migrate/full-data-from-mysql.md` which conveys a clearer meaning~

Reply:
And no need to add this file to `TOC.md`?